src/main/scala/ir/transforms/StripUnreachableFunctions.scala

```diff
@@ -47,5 +50,34 @@ def stripUnreachableFunctions(p: Program, depth: Int = Int.MaxValue): Unit = {
   }
   assert(invariant.blocksUniqueToEachProcedure(p))
   assert(invariant.cfgCorrect(p))
-
 }
+
+def getStripUnreachableFunctionsTransform(depth: Int): Transform =
+  SingleTransform(
```

Comment:
i dont really like the need to wrap parametrised transforms in a get method. maybe the Transform class can be changed to take an additional custom parameter.